postgres0.yml

```diff
@@ -19,6 +19,13 @@ restapi:
 etcd:
   host: 127.0.0.1:2379
 
+#raft:
+#  data_dir: .
+#  self_addr: 127.0.0.1:2222
+#  partner_addrs:
+#  - 127.0.0.1:2223
+#  - 127.0.0.1:2224
```

Comment:
Instead of just having a list of partner nodes, could we just list all members like we would with `etcd/hosts`? Would make it easier to manage since not every node would have a different list.

Reply:
I don't really get it. What hosts or nodes are you talking about here?
Do you mean that we should handle a situation when `self_add` is present in the `partner_addrs`?

Actually this list is just for the first start. Once you got it running Patroni will take the list from the snapshot on the filesystem: https://github.com/zalando/patroni/pull/375/files#diff-37c82fd4778c4adc14ce74a3751d3ff3R227